tests/main/interfaces-shared-memory-private/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Ensure that the shared-memory interface's private mode works.
+
+details: |
+    The shared-memory interface has a "private" attribute.
+    The shared-memory interface allows two snaps to share a POSIX shared memory
+    object declared in the slot of the provider snap.
```

Comment:
Is this part of the description a leftover? It does not seem very relevant here.